gapii/cc/BUILD.bazel

```diff
@@ -128,6 +128,7 @@ cc_library(
     copts = cc_copts() + select({
         "//tools/build:windows": ["-Wa,-mbig-obj"],
         "//conditions:default": [],
+        "//tools/build:linux": ["-Wno-error=class-memaccess"],
```

Comment:
Are these errors in the header files? It seems like we may be able to do without having it in the base bazel. If that is true, then we should only enable it in the other projects (and we can submit it there)